pkg/gameservers/migration.go

```diff
@@ -132,86 +132,109 @@ func (mc *MigrationController) loggerForGameServer(gs *agonesv1.GameServer) *log
 	return mc.loggerForGameServerKey(gsName).WithField("gs", gs)
 }
 
-// isActiveGameServerWithNode tests to see if a Pod is active and has a Node assigned.
-func isActiveGameServerWithNode(pod *corev1.Pod) bool {
-	return pod.Spec.NodeName != "" && pod.ObjectMeta.DeletionTimestamp.IsZero() && isGameServerPod(pod)
-}
+func (mc *MigrationController) isMigratingGameServerPod(pod *k8sv1.Pod) (*agonesv1.GameServer, *k8sv1.Node, bool, error) {
+	mc.baseLogger.Info("RUNNING isMigratingGameServerPod")
```

Comment:
.Debug, or delete maybe? At the very lease maybe log the pod name? (We have a lot of one-liner useless logs in the control plane already :/)

Reply:
Ah thanks - good catch, I'll make that change.

Reply:
Thanks! LGTM